roles/github/templates/run-tempest.yml.j2

```diff
@@ -79,3 +114,10 @@ jobs:
 <% if github_final_hook | length >= 1 +%>
       %% github_final_hook | indent(width=6, first=false) -%%
 <% endif %>
+
+      - name: Publish Test Report
+        uses: mikepenz/action-junit-report@v4
```

Comment:
For actions we don't control, should we consider pinning to a sha digest: https://blog.rafaelgss.dev/why-you-should-pin-actions-by-commit-hash, to guard against a malicious actor changing tags i.e tags are not immutable?

Reply:
Sure. I was thinking of doing this in another PR because of all the warnings around NodeJS 16 for existing actions being used.

I will update the PR for this action and propose another PR for the other actions.